src/modules/RA/DataLayer/components/tabs/EmbedTab/EmbedItemInput.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { SelectInput, TextInput } from 'react-admin';
+
+import { Icon } from '@blueprintjs/core';
+
+const bpIcons = [
```

Comment:
Maybe we should have this in a seperate file? A list of all usable bp icons is not a bad idea and could be used elsewhere